.github/workflows/build_test.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }} # compiles and test on Ubuntu
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: ["11"]
+      fail-fast: false
+
+    steps:
+    - name: Checkout PR branch
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
please add fetch-depth: 0 to avoid baseline errors

Reply:
Added.